src/EnergyPlus/DaylightingManager.hh

```diff
@@ -589,6 +657,60 @@ struct DaylightingManagerData : BaseGlobalStruct {
         this->TDDFluxTrans.deallocate();
         this->MapErrIndex.deallocate();
         this->RefErrIndex.deallocate();
+        this->MySunIsUpFlag = false;
+        this->CalcDayltgCoeffsMapPointsMySunIsUpFlag = false;
+        // seems like a reasonable initialization for the Vector variables
+        this->AR = 0.0;
```

Comment:
These assignments reset the three vector coefficients to zero using `a=b=c=0` assignment.  But it really doesn't make a performance difference anyway because this is in the `clear_state` function which is only called in between unit tests or in between simulations in the API.